src/sonic-daemon-base/sonic_daemon_base/daemon_base.py

```diff
@@ -41,6 +40,7 @@
 #
 
 def db_connect(db_name, namespace=EMPTY_NAMESPACE):
+    from swsscommon import swsscommon
```

Comment:
Why move this?

Reply:
This was how it was implemented originally. In my first commit tried to move to the top of the file as a common import. But tests were failing here in vsimage tests, hence I reverted to how it is used at present in repo.

00:17:46  tests/test_thermalctld.py:4: in <module>
00:17:46      from sonic_daemon_base import daemon_base
00:17:46  /usr/local/lib/python2.7/dist-packages/sonic_daemon_base/daemon_base.py:12: in <module>
00:17:46      raise ImportError (str(e) + " - required module not found")